pkg/sdk/sequences_def.go

```diff
@@ -0,0 +1,115 @@
+package sdk
+
+import g "github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk/poc/generator"
+
+//go:generate go run ./poc/main.go
+
+var sequenceSet = g.NewQueryStruct("SequenceSet").
+	PredefinedQueryStructField("ValuesBehavior", "*ValuesBehavior", g.KeywordOptions()).
+	OptionalTextAssignment("COMMENT", g.ParameterOptions().SingleQuotes())
+
+var SequencesDef = g.NewInterface(
+	"Sequences",
+	"Sequence",
+	g.KindOfT[SchemaObjectIdentifier](),
+).CreateOperation(
+	"https://docs.snowflake.com/en/sql-reference/sql/create-sequence",
+	g.NewQueryStruct("CreateSequence").
+		Create().
+		OrReplace().
+		SQL("SEQUENCE").
+		IfNotExists().
+		Name().
+		OptionalSQL("WITH").
```

Comment:
this doesn't change anything. It can be removed

Reply:
removed